src/views/Start/Start.js

```diff
@@ -108,7 +114,7 @@ function Start() {
                             let temp_dt = 'en';
                             localStorage.setItem('apphomelang', temp_dt);
                             set_sel_lang(temp_dt);
-                            //window.location.reload();
```

Comment:
window.location.reload is not required?

Reply:
no I was commented already so i removed it

Reply:
No